.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
```

Comment:
These need to be able to run in the CI pipeline. You can bypass the hooks super easily and I really dislike them so I usually never install them. Because sometimes I just want to commit a work in progress that I will fix later. 

So for now we can leave it is as is. But I will open a PR to remove this and run this through the makefile outside of hooks. Also this needs python to run (if I am not mistaken). And adding python just to run the hooks seems really strange.